_pages/ato/testing.md

```diff
@@ -0,0 +1,45 @@
+---
+title: ATO Security Testing 
+navtitle: ATO 
+---
+
+### Getting ready to test
+
+The system's technical stack needs to be relatively stable before authorization. This includes a complete list of:
+
+* AWS/Cloud.gov services required
+* base code language(s) used and their frameworks
+* third-party services, regardless of level of integration
+* all top-level URLS (ex: staging.18f.gov _and_ 18f.gsa.gov)
```

Comment:
Seems that these are all included in the [about.yml](https://github.com/18F/about_yml#aboutyml-cheat-sheet), so I think we can leave them out here.